javafx-example/src/main/java/ChatApp.java

```diff
@@ -0,0 +1,94 @@
+import javafx.application.Application;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.scene.Scene;
+import javafx.scene.control.*;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class ChatApp extends Application {
+
+    private static final Logger LOGGER = LogManager.getLogger(ChatApp.class);
+
+    private static final double WIDTH = 1500;
```

Comment:
Is there a way to not hardcode this numbers to make UI more adaptable to the screen size?

Reply:
fixed, no size is used, so the screen is resizable (not the components inside, I don't think that's needed for a demo)